### PR TITLE
Update Confirmation view alert

### LINKED ISSUE #42: [BUG] Confirmation alert timeout
**Describe the bug**

Confirmation alert keeps opened forever.

**To Reproduce**
1. Select an user.
2. Select a product.
3. Buy!
4. See confirmation alert opened forever (until "OK" is touched)

**Expected behavior**

We should automatically close the confirmation alert if the user does not close it manually.

### PR DESCRIPTION
**Pull request by izn**
## Proposed Changes

Hey ya.

This PR updates Confirmation view to render the new `InformationModal` with informative icon/text and optional timeout functionality.

### :camera:

| Success | Failure |
| ----------- | --------- |
|  |  |

### Related Issues

Fix #42.

### CODE REVIEW

File: src/views/Confirmation/Confirmation.view.tsx [hunk 1]
```diff
@@ -54,55 +64,90 @@ export class Confirmation extends Component<IConfirmationProps, IConfirmationSta
...
+    return (
+      <InformationModal
+        iconComponent={(
+          <Icon
+            name="exclamation-circle"
+            type="font-awesome"
+            color="#f44336"
+            size={40} />
+          )}
+        centerText="Whoops. Algo deu errado. :("
```
[the remark this answers is not in the document]
Comment (izn) [reply src/views/Confirmation/Confirmation.view.tsx]: I agree, but I think we could extract it later.

File: src/views/Confirmation/Confirmation.view.tsx [hunk 2]
```diff
@@ -9,12 +10,13 @@ import * as ConfirmationService from './Confirmation.service'
 import * as ConfirmationStyle from './Confirmation.style'
 
 export class Confirmation extends Component<IConfirmationProps, IConfirmationState> {
-
```
Comment (castilh0s) [inline src/views/Confirmation/Confirmation.view.tsx]: @rhogeranacleto look at this...

Comment (rhogeranacleto) [reply src/views/Confirmation/Confirmation.view.tsx]: image
image
image

File: src/components/InformationModal/InformationModal.style.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { StyleSheet } from 'react-native'
+
+export const sheet = StyleSheet.create({
+  container: {
+    flex: 1,
+    padding: 24,
+  },
+  contentContainer: {
+    flex: 1/1,
```
Comment (rhogeranacleto) [inline src/components/InformationModal/InformationModal.style.ts]: tafuck?

File: src/components/InformationModal/InformationModal.style.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  text: {
+    marginTop: 20,
+    fontSize: 20,
+    textAlign: 'center',
+  },
+  dismissButtonView: {
+    flex: 0/1,
```
Comment (rhogeranacleto) [inline src/components/InformationModal/InformationModal.style.ts]: tafuck? 2

File: src/components/InformationModal/InformationModal.style.ts (hunk 3 above)
Comment (izn) [reply src/components/InformationModal/InformationModal.style.ts]: I'm sorry. I was drunk.

File: src/components/InformationModal/InformationModal.style.ts (hunk 4 above)
Comment (izn) [reply src/components/InformationModal/InformationModal.style.ts]: I'm sorry. I was drunk.

File: src/views/Confirmation/Confirmation.view.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (millnitzluan) [reply src/views/Confirmation/Confirmation.view.tsx]: Nop,

**Pull request merged**